ckanext/versioning/tests/__init__.py

```diff
@@ -1,7 +1,14 @@
+import mock
```

Comment:
Don't mock. Abstract out the use of the GitHub backend to config and override config in test. 

As a rule of thumb: if you need to mock / monkey-patch things to test them, double check that you *really* have to do that. It's a bit of a code smell and the cases in which it is *really* needed are typically very low level stuff that just always have IO / side effects.